projects/admin/src/app/record/detail-view/item-detail-view/item-transaction/item-transaction.component.ts

```diff
@@ -113,6 +113,15 @@ export class ItemTransactionComponent implements OnInit, OnDestroy {
     private _dialogService: DialogService
   ) {}
 
+  /**
+   * Get current pickup location
+   * @return pickup location name
+   */
+  getCurrentPickupLocation(): string {
+    const location = this.pickupLocations.find(loc => loc.value === this.transaction.metadata.pickup_location_pid);
+    return location.label;
```

Comment:
What happens if no location is found. I think this case must be handled.

Reply:
Ok